### DISCUSSION TITLE
آموزش ساخت کانفیگ در پنل مرزبان

### DISCUSSION THREAD

**Discussion by alekibod** [Gozargah/Marzban#534] [General]
درود.
امکانش هست ساخت کانفیک در پنل مرزبان آموزش  بدین.
ممنون

**Comment by y8nes** [comment]
آموزش های زیادی هست میتونید با سرچ کردن پیدا کنید

**Comment by my-panel** [comment]
**✅ اگه تو ساختش مشکل دارید تلگرام پیام بدید بهم راهنماییتون میکنم دوستان :**
**https://t.me/anony_muos**

**Comment by lenzog** [comment]
Jjjw
